feature-libs/product-configurator/common/assets/translations/en/configurator-common.ts

```diff
@@ -87,6 +87,18 @@ export const configurator = {
       viewConfigurationDetails: '',
     },
     a11y: {
+      configureProduct: 'Configure product',
+      cartEntryBundleInfoSg: 'There is an item ',
+      cartEntryBundleInfoPl: 'There are {{items}} items ',
```

Comment:
Have you checked if we can do this with built-in ng plural handling?
See e.g. our resource key items_plural

Reply:
Done!